src/Public/js/nipsweb.librarypane.js

```diff
@@ -66,7 +67,7 @@ $(document).ready(function () {
       //Load a managed playlist
       $("#res-loading").show();
       $.ajax({
-        url: myradio.makeURL("MyRadio", "a-findtrack"),
+        url: mConfig.api_url + "/v2/track/search/",
```

Comment:
This may be useful to you, or may need imporving, but it's probably worth using the API functionality from one place: https://github.com/UniversityRadioYork/MyRadio/blob/master/src/Public/js/myradio.core.js#L52